### DISCUSSION TITLE
GitHub Pages 500 Error

### DISCUSSION THREAD

**Discussion by scythe7git** [community/community#87387] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello! Every time I try and load the GitHub Pages section for my repo, it gives me a 500 error. I've tried creating a new repo, and deleting my browser cache, but nothing is working. Is there anything I can do? I attached an image below:

![image](https://github.com/community/community/assets/67534127/d1c061b6-bbb1-488b-8195-935e47419334)

**Comment by yoannchaudet** [comment]
This should be the same as https://github.com/orgs/community/discussions/87404.
